tokens/src/core/components/Carousel.json

```diff
@@ -25,11 +25,20 @@
     }
   },
   "transition": {
+    "$type": "transition",
     "carousel": {
-      "base": { "source": "$carousel-transition", "$value": "transform {transition.carousel.duration} ease-in-out", "$type": "transition" },
+      "base": {
+        "source": "$carousel-transition",
+        "$value": {
+          "property": "transform",
+          "duration": "{transition.carousel.duration}",
+          "delay": "0ms",
+          "timingFunction": "ease-in-out"
+        }
+      },
       "duration": { "source": "$carousel-transition-duration", "$value": ".6s", "$type": "duration" },
-      "indicator": { "source": "$carousel-indicator-transition", "$value": "opacity .6s ease", "$type": "transition" },
-      "control": { "source": "$carousel-control-transition", "$value": "opacity .15s ease", "$type": "transition" }
+      "indicator": { "source": "$carousel-indicator-transition", "$value": "opacity .6s ease" },
+      "control": { "source": "$carousel-control-transition", "$value": "opacity .15s ease" }
```

Comment:
[curious/clarification] Are these tokens related to `transition` also composite tokens that should define the transition in their individual parts?

Something like:

```json
{
  "transition": {
    "carousel": {
      "indicator": { 
        "source": "$carousel-indicator-transition",
        "$value": {
          "property": "opacity",
          "duration": "{transition.carousel.duration}",
          "delay": 0,
          "timingFunction": "ease",
        }
      }
    }
  }
}
```

Reply:
This got me curious about other possible composite values so I did a regex search on this branch for `"\$value":\s*"(?![^"]*\s[^"]*$)([^"]*\s[^"]*)"` - it caught the `calc` ones but I also found

https://github.com/openedx/paragon/blob/372df044a9688670784a82d68a32441ecafc7b5a/tokens/src/core/components/DataTable.json#L24

https://github.com/openedx/paragon/blob/372df044a9688670784a82d68a32441ecafc7b5a/tokens/src/core/components/Dropdown.json#L40

https://github.com/openedx/paragon/blob/372df044a9688670784a82d68a32441ecafc7b5a/tokens/src/core/components/Modal.json#L30

https://github.com/openedx/paragon/blob/372df044a9688670784a82d68a32441ecafc7b5a/tokens/src/core/components/Modal.json#L34

https://github.com/openedx/paragon/blob/372df044a9688670784a82d68a32441ecafc7b5a/tokens/src/core/components/ProgressBar.json#L34

https://github.com/openedx/paragon/blob/372df044a9688670784a82d68a32441ecafc7b5a/tokens/src/core/components/ProgressBar.json#L35

https://github.com/openedx/paragon/blob/372df044a9688670784a82d68a32441ecafc7b5a/tokens/src/core/components/Form/spacing.json#L80

https://github.com/openedx/paragon/blob/372df044a9688670784a82d68a32441ecafc7b5a/tokens/src/core/components/Form/transition.json#L7

https://github.com/openedx/paragon/blob/372df044a9688670784a82d68a32441ecafc7b5a/tokens/src/core/components/Form/transition.json#L11

https://github.com/openedx/paragon/blob/372df044a9688670784a82d68a32441ecafc7b5a/tokens/src/core/global/transition.json#L6

https://github.com/openedx/paragon/blob/372df044a9688670784a82d68a32441ecafc7b5a/tokens/src/core/global/transition.json#L11

https://github.com/openedx/paragon/blob/372df044a9688670784a82d68a32441ecafc7b5a/tokens/src/core/global/transition.json#L17

https://github.com/openedx/paragon/blob/372df044a9688670784a82d68a32441ecafc7b5a/tokens/src/core/global/transition.json#L22

https://github.com/openedx/paragon/blob/372df044a9688670784a82d68a32441ecafc7b5a/tokens/src/core/global/typography.json#L14

https://github.com/openedx/paragon/blob/372df044a9688670784a82d68a32441ecafc7b5a/tokens/src/core/global/typography.json#L25

https://github.com/openedx/paragon/blob/372df044a9688670784a82d68a32441ecafc7b5a/tokens/src/themes/light/components/Nav.json#L10

https://github.com/openedx/paragon/blob/372df044a9688670784a82d68a32441ecafc7b5a/tokens/src/themes/light/components/Form/other.json#L102

I don't think all of these are problematic but it'd be good to think through which ones are OK and which ones need to be updated.

Reply:
> [curious/clarification] Are these tokens related to transition also composite tokens that should define the transition in their individual parts?

Yes, thanks
Added additional composite values

> This got me curious about other possible composite values so I did a regex search on this branch for "\$value":\s*"(?![^"]*\s[^"]*$)([^"]*\s[^"]*)" - it caught the calc ones but I also found

Thank you for providing this list, now we will definitely not miss anything! 💯 

> Find where we're using the consolidated token
`"header": { "source": "$dropdown-header-padding", "$value": "{spacing.dropdown.padding.y.base} {spacing.dropdown.padding.x.item}" } `

This variable came to us from bootstrap 4, during the creation of tokens we transferred [the file](https://github.com/openedx/paragon/blob/Peter_Kulko/expanded-tokens-value/src/Dropdown/dropdown-bootstrap.scss#L171) from bootstrap to determine its value from Paragon.

> Not sure... sidenote: does this need a calc?
"$value": "{other.content.form.control.select.indicator.icon} no-repeat right {spacing.form.input.padding.y.base} center / {color.form.control.select.bg.base}", 

`/` is a part of the syntax that allows you to specify both the position of the image and its size at the same time (it is not a mathematical operator).